features/support/minitest_disable.rb

```diff
@@ -0,0 +1,2 @@
+require 'multi_test'
+MultiTest.disable_autorun
```

Comment:
https://github.com/cucumber/multi_test/pull/2#issuecomment-21863459

Using this for the time being until we can find a better solution.